test/trim.js

```diff
@@ -3,7 +3,7 @@ var eq = require('./shared/eq');
 
 
 describe('trim', function() {
-  var test = '\x09\x0A\x0B\x0C\x0D\x20\xA0\u1680\u180E\u2000\u2001\u2002\u2003\u2004\u2005\u2006\u2007\u2008\u2009\u200A\u202F\u205F\u3000\u2028\u2029\uFEFFHello, World!\x09\x0A\x0B\x0C\x0D\x20\xA0\u1680\u180E\u2000\u2001\u2002\u2003\u2004\u2005\u2006\u2007\u2008\u2009\u200A\u202F\u205F\u3000\u2028\u2029\uFEFF';
```

Comment:
The `\u180E` character was removed in a previous pull request but the tests were not updated at the same time. This is now fixed.